eliot/tests/test_testing.py

```diff
@@ -437,13 +440,13 @@ def test_raises(self):
         """
         exc = Exception()
         class MyTest(TestCase):
-            @validateLogging(None)
+            @self.validate(None)
             def test_foo(self, logger):
                 raise exc
 
         raised = None
         try:
-            MyTest("test_foo").test_foo()
+            MyTest("test_foo").debug()
```

Comment:
I think beforehand no cleanup was necessary; now if you don't cleanup then you have the global logger overriden.